Sources/DatadogSDKTesting/IntelligentTestRunner/IntelligentTestRunner.swift

```diff
@@ -10,18 +10,30 @@ import Foundation
 class IntelligentTestRunner {
     private let itrCachePath = "itr"
     private let skippableFileName = "skippableTests"
-    private var _skippableTests: SkipTests? = nil
+    private var _skippableTests: SkipTests? = nil {
+        didSet {
+            skippableTests.removeAll()
+            guard let newTests = _skippableTests else { return }
+            for test in newTests.tests {
+                skippableTests.get(key: test.suite, or: [:]) { methods in
+                    methods.get(key: test.name, or: []) { $0.append(test) }
+                }
+            }
+        }
+    }
 
     var configurations: [String: String]
     var itrFolder: Directory?
+
+    // [suite: [name: [info]]]
```

Comment:
Maybe worth a dedicated model at this point?

Reply:
yep. Changed to the model